io.github.MakovWait.Godots.json

```diff
@@ -0,0 +1,108 @@
+{
+    "id" : "io.github.MakovWait.Godots",
+    "runtime" : "org.freedesktop.Sdk",
+    "runtime-version" : "23.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11",
+        "org.freedesktop.Sdk.Extension.dotnet7"
+    ],
+    "command": "godots",
+    "build-options": {
+        "append-path": "/usr/lib/sdk/dotnet7/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet7/lib",
+        "env": {
+            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig"
+        }
+    },
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=x11",
+        "--device=all",
+        "--socket=pulseaudio",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--env=DOTNET_ROOT=/app/lib/dotnet",
+        "--env=DOTNET_CLI_TELEMETRY_OPTOUT=true",
+        "--env=DOTNET_NOLOGO=true"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/openjdk11/install.sh"
+            ]
+        },
+        {
+            "name": "dotnet",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/dotnet7/bin/install-sdk.sh",
+                "cp -r /usr/lib/sdk/dotnet7/lib/packs/ /app/lib/dotnet/"
+            ]
+        },
+        {
+            "name" : "Godots",
+            "builddir" : true,
+            "buildsystem" : "simple",
+            "build-commands": [
+                "unzip Godot_v4.1.1-stable_linux.x86_64.zip",
```

Comment:
so how does this work on aarch64?

You still build an aarch64 flatpak here.

Reply:
I don't think this works on aarch64 at the moment. I should probably have it skip aarch64 then?